style.css

```diff
@@ -398,3 +398,110 @@ span {
   background: #fff;
   margin-bottom: 30px;
 }
+
+.desktop-tools {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-link {
+    display: none;
+  }
+
+  .nav-container {
+    padding: 10px 40px;
+  }
+
+  .desktop-tools {
+    display: inline;
+  }
+
+  .display-tools,
+  li,
+  a {
+    padding-right: 10px;
+    height: 0%;
+  }
+
+  .major-container {
+    padding: 40px 50px;
+  }
+
+  .project-card {
+    display: grid;
+    padding: 10px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
```

Comment:
- Hey, please, let's adjust our on cards so they have some spaces left and right and not taking the whole width of this project section thereby making it close to the Figma design as much as you can below:


*your design*
![Screenshot from 2022-12-16 17-08-19](https://user-images.githubusercontent.com/59397159/208140849-d12fb7d2-0437-4955-8e30-c09936f30b9a.png)

*Figma design*
![Screenshot from 2022-12-16 17-07-35](https://user-images.githubusercontent.com/59397159/208140857-21677729-4fa4-4e85-8228-8c95c398d0b6.png)